common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -316,6 +316,16 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionOrderByOldest = Switch(
+    SwitchGroup.Feature,
+    "discussion-order-by-oldest",
+    "If on, comments default sort ordering will be oldest first. If off, then the default is newest first",
+    owners = Seq(Owner.withGithub("nicl")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
+  )
+
```

Comment:
With this switch - is the intention of this to make a final decision?
If so - it might be worth making it a test switch so that we don't permanently add this complexity to the code.